manifest.json

```diff
@@ -7,15 +7,15 @@
         "fr": "Un forum de nouvelle génération, simplement.",
         "de": "Forum der nächsten Generation leicht gemacht."
     },
-    "version": "0.1.0.14~ynh2",
+    "version": "0.1.0.14~ynh3",
     "url": "http://flarum.org/",
     "license": "MIT",
     "maintainer": {
         "name": "tituspijean",
         "email": "tituspijean@outlook.com"
     },
     "requirements": {
-        "yunohost": ">= 3.8.1"
+        "yunohost": ">= 4.1"
```

Comment:
... and bump requirements to 4.1.2 if you are using `ynh_legacy_permissions_delete_all` https://github.com/YunoHost/yunohost/blob/adc83b4c9c2c30e9ef75f3609c538b646f91f1db/data/helpers.d/permission#L398